OJS.Workers.ExecutionStrategies/Python/PythonDjangoOrmExecutionStrategy.cs

```diff
@@ -142,12 +148,12 @@ private void RestorePackages(IExecutor executor, IExecutionContext<TestsInputMod
             throw new ArgumentException($"Failed to restore packages! " + this.GetErrorOutput(result));
         }
 
-        private void CreateVirtualEnvironment(IExecutor executor, IExecutionContext<TestsInputModel> executionContext)
+        private void CreateVirtualEnvironment(IExecutor executor, IExecutionContext<TestsInputModel> executionContext, string envName)
         {
-            this.DeleteVirtualEnvironment(executor, executionContext);
+            this.DeleteVirtualEnvironment(executor, executionContext, envName);
```

Comment:
This is redundant now, since the `envName` will be unique every time and no env will be deleted at this point.

Reply:
Correct. It is fixed